interfaces/builtin/network_manager.go

```diff
@@ -257,6 +257,13 @@ dbus (receive, send)
     path=/org/freedesktop/NetworkManager{,/**}
     peer=(label=###PLUG_SECURITY_TAGS###),
 
+# NM implements org.freedesktop.DBus.ObjectManager too
```

Comment:
Can you modify these comments to be:
```
# Later versions of NetworkManager implement org.freedesktop.DBus.ObjectManager for
# clients to easily obtain all (and be alerted to added/removed) objects from the service.
```

Reply:
I did this now, once spread is green I will merge.